eth/handler.go

```diff
@@ -272,11 +268,7 @@ func newHandler(config *handlerConfig) (*handler, error) {
 			}
 			return 0, nil
 		}
-		n, err := h.chain.InsertChain(blocks)
-		if err == nil {
-			h.enableSyncedFeatures() // Mark initial sync done on any fetcher import
-		}
-		return n, err
```

Comment:
The node will only be marked as initialized after the initial sync cycle. There are some behavioral changes in legacy un-merged chain, but it's fine, just do a tiny full-sync instead of a single block fetch.

Reply:
What happens with empty networks that are just starting up? They can't do any type of sync because there's nothing yet to sync

Reply:
Ah this is specifically the fetcher path. Still in that case, what happens if my node never does a sync cycle because it's always in sync and connected?